scripts/cve-parsing/write-dates2-cves.py

```diff
@@ -0,0 +1,30 @@
+import sys, csv
```

Comment:
This file shouldn't have been edited.

Reply:
I think it was my editor added stuff. It was un intentional